spec/controllers/users/verify_profile_controller_spec.rb

```diff
@@ -1,28 +1,67 @@
 require 'rails_helper'
 
-RSpec.describe Users::VerifyProfileController do
+describe Users::VerifyProfileController do
```

Comment:
I think @zachmargolis likes it when we keep this `RSpec` for good reasons that I myself cannot articulate on this fine Friday afternoon